index.js

```diff
@@ -251,8 +251,17 @@ exports.onMessage = function(msg, dobby) {
 
             break;
         case '.qst':
-            var d = new Date().toLocaleTimeString().toString();
-            dobby.respond(d)
+            function hour12 (hour) {
+                var mod = hour % 12;
+
+                if (mod < 10) {
+                    return '0'.concat(mod);
+                } else {
+                    return mod;
+                }
+            }
+            var date = new Date()
+            dobby.respond(new Date().toLocaleTimeString().replace(/[\d]+/, hour12(date.getHours())));
```

Comment:
replace `new Date()` on this line with `date` (we already instantiated it the line above).